renderdoc/CMakeLists.txt

```diff
@@ -139,6 +142,7 @@ set(sources
     core/intervals_tests.cpp
     core/bit_flag_iterator.h
     core/bit_flag_iterator_tests.cpp
+    core/bit_flag_iterator.h
```

Comment:
This file is already listed a couple of lines up?

Reply:
Thank you for noticing